dhcp.nix

```
@@ -3,11 +3,81 @@ let
   cfg = config.pi-router;
 in
 {
+  systemd.network.enable = true;
 
-  # TODO Setup DHCP. Maybe KEA?
+  networking = {
+    useDHCP = false;
+    nameservers = cfg.dnsServers;
+    firewall.interfaces."${cfg.interfaces.lan.name}".allowedUDPPorts = [ 67 68 ];
```

I'm pretty sure we only have to open UDP port 67.
Can we try that?

---

Can confirm - still works:

```suggestion
    firewall.interfaces."${cfg.interfaces.lan.name}".allowedUDPPorts = [ 67 ];
```